packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -177,18 +179,25 @@ EOF
 %{foreman_proxy_bundlerd_dir}/%{plugin_name}.rb
 %exclude %{gem_cache}
 %{gem_spec}
-%{foreman_proxy_bundlerd_dir}/smart_proxy_salt_core.rb
-%{smart_proxy_dynflow_bundlerd_dir}/smart_proxy_salt_core.rb
 %config(noreplace) %{salt_config_dir}/foreman.yaml
 %config %{_root_sysconfdir}/cron.d/%{gem_name}
 %exclude %{gem_instdir}/etc
 %exclude %{gem_instdir}/cron
+%attr(-,foreman-proxy,foreman-proxy) %{salt_proxy_runners_dir}/foreman_file.py*
+%attr(-,foreman-proxy,foreman-proxy) %{salt_proxy_runners_dir}/foreman_https.py*
+%attr(-,foreman-proxy,foreman-proxy) %{salt_proxy_reactors_dir}/foreman_minion_auth.sls
```

Comment:
why do these files need to be owned by `foreman-proxy`?

Reply:
I checked it and it's actually not necessary. Should these lines be left out completely, then?

Reply:
yes please

Reply:
@bastian-src you *do* need to include those dirs, just no need to set the permissions